bin/improver-nbhood

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
I'm satisfied with this, although it may be revisited in the future when these future usages become clearer.